lib/kuniri/core/setting.rb

```diff
@@ -20,10 +20,9 @@ def Setting.create
         @@settings = new unless @@settings
         return @@settings
       end
-
+      
```

Comment:
Sorry about this, I deletes the comment bellow and I put back again thinking it would be useful